Snippets/Push/01-push-notifications.swift

```diff
@@ -0,0 +1,144 @@
+    publishKey: "demo",
+    subscribeKey: "demo",
+    userId: "myUniqueUserId"
+  )
+)
+
+// snippet.end
+
+// snippet.add-apns-devices-on-channels
+// Adding a device to a channel for APNs
+pubnub.addAPNSDevicesOnChannels(
+  ["channelSwift"],
+  device: Data([0x01, 0x02, 0x03, 0x04]), // Replace with actual device token
+  on: "com.app.bundle",
+  environment: .production
+) { result in
+  switch result {
+  case let .success(channels):
+    print("The list of channels added for push: \(channels)")
+  case let .failure(error):
+    print("Failed Push List Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.list-apns-push-channel-registrations
+pubnub.listAPNSPushChannelRegistrations(
+  for: Data([0x01, 0x02, 0x03, 0x04]), // Replace with actual device token
+  on: "com.app.bundle",
+  environment: .production
+) { result in
+  switch result {
+  case let .success(channels):
+    print("The list of channels enabled for push: \(channels)")
+  case let .failure(error):
+    print("Failed Push List Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.remove-apns-devices-on-channels
+pubnub.removeAPNSDevicesOnChannels(
+  ["channelSwift"],
+  device: Data([0x01, 0x02, 0x03, 0x04]), // Replace with actual device token
+  on: "com.app.bundle",
+  environment: .production
+) { result in
+  switch result {
+  case let .success(channels):
+    print("The list of channels disabled for push: \(channels)")
+  case let .failure(error):
+    print("Failed Push List Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.remove-all-apns-push-device
+pubnub.removeAllAPNSPushDevice(
+  for: Data([0x01, 0x02, 0x03, 0x04]), // Replace with actual device token
+  on: "com.app.bundle",
+  environment: .production
+) { result in
+  switch result {
+  case .success:
+    print("All channels have been removed for the device token.")
+  case let .failure(error):
+    print("Failed Push List Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.add-push-channels-registrations
+pubnub.addPushChannelRegistrations(
+  ["channelSwift"],
+  for: Data([0x01, 0x02, 0x03, 0x04]) // Replace with actual device token
+) { result in
+  switch result {
+  case let .success(channels):
+    print("The list of channels added for push: \(channels)")
+  case let .failure(error):
+    print("Failed Push Modification Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.list-push-channels-registrations
+pubnub.listPushChannelRegistrations(
+  for: Data([0x01, 0x02, 0x03, 0x04]) // Replace with actual device token
+) { result in
+  switch result {
+  case let .success(channels):
+    print("The list of channels enabled for push: \(channels)")
+  case let .failure(error):
+    print("Failed Push List Response: \(error.localizedDescription)")
+  }
+}
+// snippet.end
+
+// snippet.remove-push-channel-registrations
+pubnub.removePushChannelRegistrations(
```

Comment:
Just randomly picking method call because there is a lot of them across all files for same reason: there is no description of the action which is present in some other methods call.

Reply:
I think I will remove those descriptions, because any comments inside them may duplicate what was actually written in the docs